src/status_im/subs.cljs

```diff
@@ -1633,3 +1634,27 @@
  :<- [:search/filter]
  (fn [[chats search-filter]]
    (apply-filter search-filter chats extract-chat-attributes)))
+
+;;ENS ==================================================================================================================
+
+(re-frame/reg-sub
+ :ens/state
+ (fn [db]
+   (get-in db [:ens :state])))
+
+(re-frame/reg-sub
+ :ens/username
+ (fn [db]
+   (get-in db [:ens :username])))
```

Comment:
`:ens` looks like too general, maybe we could use something like `:ens-usernames`

Reply:
I think :ens is fine